src/tree/updater_gpu_hist.cu

```diff
@@ -270,75 +320,119 @@ struct DeviceShard {
 
   dh::CubMemory temp_memory;
 
+  // TODO(canonizer): do add support multi-batch DMatrix here
```

Comment:
Unnecessary TODO based on our conversation about removing support for external memory?

Reply:
Done.